lib/src/sdk.ts

```diff
@@ -1,14 +1,12 @@
 import {
   Account,
-  CardanoTransaction,
-  CryptoCurrency,
-  Currency,
   ElrondTransaction,
-  NearTransaction,
-  NeoTransaction,
   RippleTransaction,
```

Comment:
This library is common for any kind of exchange, not sure why we are removing some families.

Reply:
Not used in the file, I auto removed not used imports